### DISCUSSION TITLE
It is possible to use single letters to replace "micro" with "u", "pico" with "p", "nano" with "n".

### DISCUSSION THREAD

**Discussion by boromyr** [Qalculate/qalculate-gtk#503] [Q&A] [answered]
I would like to be able to use single letters as suffixes for orders of magnitude, so that "2u" corresponds to "2micro". It already seems to work for "k" as "*1000" and "M" as "*10E6"

**Comment by hanna-kn** [comment]
This is not possible because of conflicts with other objects (e.g. u is used as symbol for the atomic mass unit). 

You can however add variables with the corresponding letters and values, e.g. a variable with the name "u" and value 1E-6. This will cause any conflicting objects to be deactivated.

**Comment by boromyr** [comment]
How can I add these custom variables?

**Reply by hanna-kn** [reply, marked as answer]
Use File->New->Variable, "STO" button, Ctrl+S, the `save()` function (e.g. `save(1e-6, u, Suffixes)`), or an expression such as `u=1e-6` (creates a temporary variable which will be removed on exit).

**Reply by boromyr** [reply]
Thanks it worked
